utils/docs/lepton-tragesym.1.in

```diff
@@ -25,10 +26,166 @@ swap words in the pinlabel attributes if requested (only for
 attributes of the right and top pins, in the latter case only
 if rotation is also requested);
 .IP \(bu
-do some syntax check to the input file.
+do some syntax checking to the input file.
+
+.SH SOURCE FILE FORMAT
+.SS General
+
+Source file is a text file consisting of three sections:
+.br
+.rs
+\(bu
+.B [geda_attr]
+.br
+.rs
+\(bu
+.B [options]
+.br
+.rs
+\(bu
+.B [pins]
+.RE
+
+Section name should be enclosed in square brackets.
+
+Empty lines, lines consisting of whitespaces only, as well as
+lines beginning with the character '#' (comments) are silently
+ignored.
+
+.SS [options]
+The section contains export settings.  The following settings are
+supported:
+
+'IP \(bu
+.B wordswap
+.RI ( boolean )
+.br
+Swap labels if the pin is on the right side and contains space
+between words, that is, looks like this: \*[lq]PB1 (CLK)\*[rq].
+That may be useful for micro controller port labels.
+'IP \(bu
+.B rotate_labels
+.RI ( boolean )
+.br
+Rotate the \*[lq]pinlabel\*[rq] attribute of the top and bottom pins
+by 90 degrees.
+This may be useful for large symbols like FPGAs with more than 100
+pins.
+'IP \(bu
+.B sort_labels
+.RI ( boolean )
+.br
+Sort the pins by their \*[lq]pinlabel\*[rq] attributes, which is
+useful for address ports, busses, etc.
+'IP \(bu
+.B generate_pinseq
```

Comment:
I would elaborate a bit the `generate_pinseq` description,
saying that numbers will be generated in sequence of pin
appearance in the `[pins] section.